### ISSUE TITLE
Can't talk to Millonce for progressing Curious Curio 

### ISSUE THREAD

**Issue by Listerw** [ForgottenGlory/Masterstroke#62]
**Masterstroke Version**
Beta 2

**Describe the bug**
Can't talk to Millonce for progressing Curious Curio (Toy Story quest)

**To Reproduce**
Progress through Toy Story's second quest until you need to meet Millonce.
Enter the cellar that Millonce is in for the quest. 
Try talking to her.
Get "You're not supposed to be in here" message.

**Expected behavior**
You can talk to Millonce.

**Screenshots**
![image](https://user-images.githubusercontent.com/33641061/156893062-cc0515cb-802e-43bc-a770-2ffcb2742211.png)

**Additional context**
Save: 
[saves.zip](https://github.com/ForgottenGlory/Masterstroke/files/8191093/saves.zip)

**Comment by aloysiab** [comment]
Something deeper in the load order is changing the public flag for that area.
Someone else had an issue like this and a different mod so there was mention of it in the forum thread on LL.

"I found Millonce in the Mor Khazgur cellar, but it seems like I'm considered trespassing in there and she just tells me I'm not supposed to be in there.  I did some investigation and it looks like it's a mod conflict with Sounds of Skyrim - Toy Story edits the cell to have the public flag, but Sounds of Skyrim doesn't.  It was an easy fix with SSEEdit."

I used SSEEdit and took a look and it looks like Lux.esp which is loaded later is the one overwriting the public tag that toystory.esp which is earlier in the load order sets for that cell. SkyRem-Iris.esp also removes the public area flag but that is loaded earlier so shouldnt be an issue.
